src/validation.cpp

```diff
@@ -1704,6 +1704,7 @@ static bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockInd
     // before the first had been spent.  Since those coinbases are sufficiently buried its no longer possible to create further
     // duplicate transactions descending from the known pairs either.
     // If we're on the known chain at height greater than where BIP34 activated, we can save the db accesses needed for the BIP30 check.
+    assert(pindex->pprev && "pindex->pprev != nullptr assumed.");
```

Comment:
Repeating the condition in a message is not very useful.

Reply:
Agree with @sipa, same thing as #11133